data/config/inaira-fp.json

```diff
@@ -0,0 +1,114 @@
+[
+    {
+        "fr_setup": {
+            "fr_ready_cnxn":"tcp://127.0.0.1:5001",
+            "fr_release_cnxn":"tcp://127.0.0.1:5002"
+        },
+    "meta_endpoint":"tcp://*:5008"
+    },
+    {
+        "plugin": {
+            "load": {
+                "index":"learning",
+                "name":"InairaMLPlugin",
+                "library":"./lib/libInairaMLPlugin.so"
+            }
+        }
+    },
+    {
+        "plugin": {
+            "load": {
+                "index":"liveview",
+                "name":"LiveViewPlugin",
+                "library":"./lib/libLiveViewPlugin.so"
+            }
+        }
+    },
+    {
+        "plugin": {
+            "load": {
+                "index":"hdf",
+                "name":"FileWriterPlugin",
+                "library":"./lib/libHdf5Plugin.so"
+            }
+        }
+    },
+    {
+        "plugin": {
+            "connect": {
+                "index":"learning",
+                "connection":"frame_receiver"
+            }
+        }
+    },
+    {
+        "plugin": {
+            "connect": {
+                "index":"liveview",
+                "connection":"learning"
+            }
+        }
+    },
+    {
+        "plugin": {
+            "connect": {
+                "index":"hdf",
+                "connection":"learning"
+            }
+        }
+    },
+    {
+        "learning": {
+            "model_path":"/aeg_sw/work/projects/inaira/casting/models/casting_prob_model/",
```

Comment:
Config exposing details of our systems - committed code should have example paths and be customised once installed.